### PR TITLE
Add screen casts to release notes

### PR DESCRIPTION
**Pull request by kaviththiranga**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by SumayyaZiyad — APPROVED** [review]

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 1]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
 
 Furthermore, the extension enables you to create various types of components including RESTful APIs, GraphQL APIs, webhooks, or scheduled apps. With the extension, you can create these new components, implement the necessary logic, and deploy them to Choreo easily from within the VS Code IDE itself.
 
+### Signing In
+
+<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:13]:
```suggestion
<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>

```
We need to give the absolute (complete) path when adding GIfs or images. This is due to a rendering problem of MKDocs.

**Review by gigara — APPROVED** [review]

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 2]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
 
 Furthermore, the extension enables you to create various types of components including RESTful APIs, GraphQL APIs, webhooks, or scheduled apps. With the extension, you can create these new components, implement the necessary logic, and deploy them to Choreo easily from within the VS Code IDE itself.
 
+### Signing In
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:11]:
```suggestion
### Sign in

```

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 3]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
 
 Furthermore, the extension enables you to create various types of components including RESTful APIs, GraphQL APIs, webhooks, or scheduled apps. With the extension, you can create these new components, implement the necessary logic, and deploy them to Choreo easily from within the VS Code IDE itself.
 
+### Signing In
+
+<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>
+
+### Creating Projects
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:15]:
```suggestion
### Create project

```

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 4]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
...
 
+### Signing In
+
+<img src="./v1.0.0-screens/sign-in.gif" class="cInlineImage-full"/>
+
+### Creating Projects
+
+<img src="./v1.0.0-screens/create-project.gif" class="cInlineImage-full"/>
+
+### Cloning Projects
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:19]:
```suggestion
### Clone projects

```

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 5]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
...
+
+<img src="./v1.0.0-screens/clone-project.gif" class="cInlineImage-full"/>
+
+### Creating Components
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:23]:
```suggestion
### Create components

```

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 6]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
...
+
+<img src="./v1.0.0-screens/create-gql-api.gif" class="cInlineImage-full"/>
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:25]:
```suggestion
<img src="./v1.0.0-screens/create-gql-api.gif" class="cInlineImage-full"/>

```
Should we add descriptions to these?

File: docs/choreo-extension/release-notes/version-1.0.0.md [hunk 7]
```diff
@@ -8,4 +8,43 @@ Another significant enhancement is the ability to log in to the Choreo platform
...
+
+### Architecture View
+
+<img src="./v1.0.0-screens/linking.gif" class="cInlineImage-full"/>
+
+<img src="./v1.0.0-screens/service-design.gif" class="cInlineImage-full"/>
+
+<img src="./v1.0.0-screens/source-navigation.gif" class="cInlineImage-full"/>
+
+### Pushing to Choreo
```
Comment (praneesha) [inline docs/choreo-extension/release-notes/version-1.0.0.md:41]:
```suggestion
### Push to Choreo

```
Should it be `push` or `publish`?

**Review by praneesha — CHANGES_REQUESTED** [review]

**Review by praneesha — APPROVED** [review]

**Comment by praneesha** [comment]
Merging to address the review comments in another PR.

File: docs/choreo-extension/release-notes/version-1.0.0.md (hunk 1 above)
Comment (praneesha) [reply docs/choreo-extension/release-notes/version-1.0.0.md:13]: This should be fine as we are not displaying the release note on the website.

File: docs/choreo-extension/release-notes/version-1.0.0.md (hunk 3 above)
Comment (praneesha) [reply docs/choreo-extension/release-notes/version-1.0.0.md:15]: Done via https://github.com/wso2/choreo-vscode/pull/6.

File: docs/choreo-extension/release-notes/version-1.0.0.md (hunk 4 above)
Comment (praneesha) [reply docs/choreo-extension/release-notes/version-1.0.0.md:19]: Done via https://github.com/wso2/choreo-vscode/pull/6.

File: docs/choreo-extension/release-notes/version-1.0.0.md (hunk 5 above)
Comment (praneesha) [reply docs/choreo-extension/release-notes/version-1.0.0.md:23]: Done via https://github.com/wso2/choreo-vscode/pull/6.

File: docs/choreo-extension/release-notes/version-1.0.0.md (hunk 7 above)
Comment (praneesha) [reply docs/choreo-extension/release-notes/version-1.0.0.md:41]: Done via https://github.com/wso2/choreo-vscode/pull/6.

**Comment by praneesha** [comment]
Above review comments are addressed via Done via https://github.com/wso2/choreo-vscode/pull/6..

**Pull request merged**